src/app/login/[[...login]]/page.tsx

```
@@ -1,9 +1,93 @@
+            <div className="mt-6 grid grid-cols-1 gap-3">
+              <div className="rounded-md bg-blue-50 p-4">
+                <div className="flex">
+                  <div className="flex-shrink-0">
+                    <Lock className="h-5 w-5 text-blue-400" />
+                  </div>
+                  <div className="ml-3">
+                    <h3 className="text-sm font-medium text-blue-800">
+                      Enterprise Security
+                    </h3>
+                    <div className="mt-2 text-sm text-blue-700">
+                      <p>Access is managed through Okta's Identity Provider</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)